R4ML/R/ml.model.pca.R

```diff
@@ -30,24 +30,24 @@ setClass(
 
 #' @name r4ml.pca
 #' @title Principal Component Analysis (PCA)
-#' @description Builds a Principal Component Analysis (PCA) model on a given r4ml.matrix, allows to project
+#' @description Builds a Principal Component Analysis (PCA) model on a given r4ml.matrix, allows one to project
 #' a given dataset into the new feature space.
 #' @details r4ml.pca() computes the eigen vectors, eigen values, and square roots of
 #' the eigen values on the given dataset.
-#' @param data (r4ml.matrix) the dataset which PCA will be applied to
-#' @param k (integer) the number of dimensions in the new feature space
-#' @param center (logical) a boolean value indicating whether all columns in the data should be centered to have zero mean.
+#' @param data (r4ml.matrix):  The dataset which PCA will be applied to
+#' @param k (integer):  The number of dimensions in the new feature space
+#' @param center (logical):  A boolean value indicating whether all columns in the data should be centered to have zero mean.
 #' Default is set to FALSE.
-#' @param scale (logical) a boolean value indicating whether all columns in the data should be scaled to have variance one.
+#' @param scale (logical):  A boolean value indicating whether all columns in the data should be scaled to have variance one.
 #' Default is set to FALSE.
-#' @param projData (logical) a boolean value indicating whether the data should be projected in the new feature space. If
+#' @param projData (logical):  A boolean value indicating whether the data should be projected in the new feature space. If
 #' set to TRUE, a list (r4ml.pca model, r4ml.matrix projData) is returned. If set to FALSE, only the r4ml.pca model is returned.
 #' Default value for \code{projData} is TRUE.
-#' @param applyPCA (r4ml.pca) (optional) Currently is not supported. In future this option will import an existing r4ml.pca model. If provided, \code{data} will be projected to the
+#' @param (optional) applyPCA (r4ml.pca):  Currently is not supported. In future this option will import an existing r4ml.pca model. If provided, \code{data} will be projected to the
```

Comment:
I remember why I moved this...we kind of converged on listing the data type after the variable.  I moved (optional) so that it doesn't look like there is a data type called "optional".